sources/packages/backend/apps/db-migrations/src/sql/Reports/Update-offering-status-offering-details-report.sql

```diff
@@ -0,0 +1,74 @@
+      NOT education_programs_offerings.lacks_study_breaks AS "Has Study Breaks",
+      education_programs_offerings.actual_tuition_costs AS "Actual Tuition",
+      education_programs_offerings.program_related_costs AS "Program Related Costs",
+      education_programs_offerings.mandatory_fees AS "Mandatory Fees",
+      education_programs_offerings.exceptional_expenses AS "Exceptional Expenses",
+      education_programs_offerings.offering_type AS "Offering Type",
+      education_programs_offerings.offering_status AS "Status",
+      education_programs_offerings.study_breaks ->> ''totalFundedWeeks'' AS "Funded Weeks",
+      offerings_with_applications.total_applications AS "Total Applications"
+    FROM
+      sims.education_programs_offerings education_programs_offerings
+    INNER JOIN sims.institution_locations institution_locations ON
+      education_programs_offerings.location_id = institution_locations.id
+    INNER JOIN sims.education_programs education_programs ON
+      education_programs_offerings.program_id = education_programs.id
+    INNER JOIN sims.program_years ON
+      -- program year is used to merge the offerings and the program year table on the
+      -- provided program year which is then used to filter the offerings belonging to
+      -- the provided program year in the WHERE clause.  
+      program_years.id = :programYear
+    LEFT JOIN (
+      -- Get all the offerings having at least one application in the given 
+      -- Program year with their applications count satisfying the conditions
+      -- from the WHERE clause.
+      SELECT
+        education_programs_offerings.id AS "id",
+        COUNT(applications.id) AS "total_applications"
+      FROM
+        sims.education_programs_offerings education_programs_offerings
+      INNER JOIN sims.institution_locations institution_locations ON
+        education_programs_offerings.location_id = institution_locations.id
+      INNER JOIN sims.student_assessments student_assessments ON
+        education_programs_offerings.id = student_assessments.offering_id
+      INNER JOIN sims.applications applications ON
+        applications.current_assessment_id = student_assessments.id
+      WHERE
+        applications.program_year_id = :programYear
+        AND education_programs_offerings.offering_intensity = ANY(:offeringIntensity)
+          AND institution_locations.institution_id = :institutionId
+          AND applications.application_status = ''Completed''
+        GROUP BY
+          education_programs_offerings.id) offerings_with_applications ON
+      education_programs_offerings.id = offerings_with_applications."id"
+    WHERE
+      education_programs_offerings.offering_intensity = ANY(:offeringIntensity)
+      AND education_programs_offerings.offering_type != ''Scholastic standing''
+      and education_programs_offerings."offering_status" in (''Approved'', ''Creation pending'', ''Change under review'')
```

Comment:
Please keep the same case and `offering_status` does not need the quotes as we did for the other ones.
```sql
AND education_programs_offerings.offering_status in (''Approved'', ''Creation pending'', ''Change under review'')
```